docs/output.md

```diff
@@ -432,6 +432,44 @@ Besides the reference files or output files created by BUSCO, the following summ
   - `busco_summary.tsv`: A summary table of the BUSCO results, with % of marker genes found. If run in automated lineage selection mode, both the results for the selected domain and for the selected more specific lineage will be given, if available.
 
 </details>
+
+### QC for metagenome assembled genomes with CheckM
+
+[CheckM](https://ecogenomics.github.io/CheckM/) CheckM provides a set of tools for assessing the quality of genomes recovered from isolates, single cells, or metagenomes. It provides robust estimates of genome completeness and contamination by using collocated sets of genes that are ubiquitous and single-copy within a phylogenetic lineage
+
+By default, nf-core/mag runs CheckM with the `check_lineage` workflow that places genome bins on a reference tree to define lineage-marker sets, to check for completeness and contamination based on lineage-specific marker genes. and then subsequently runs `qa` to generate the summary files.
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `GenomeBinning/QC/CheckM/`
+  - `[assembler]-[binner]-[sample/group].txt`: Detailed statistics about bins informing completeness and contamamination scores (output of `checkm qa`).
+  - `[assembler]-[binner]-[sample/group].tsv`: Overall summary file for completeness and contamination (output of `checkm lineage_wf`).
+  - `[assembler]-[binner]-[sample/group]/`: intermediate files for CheckM results, including CheckM generated annotations, log, lineage markers etc.
+
+</details>
+
+If the parameter `--save_busco_reference` is set, additionally the used BUSCO lineage datasets are stored in the output directy.
```

Comment:
This is still redundant to above, but maybe still WIP

Reply:
Yes, I was going to remove it but then you requests having the database download - so I left it here o as not to forget :)

Reply:
Fixed! https://github.com/nf-core/mag/pull/350/commits/fd53174521a50be17e9126d42d80b3d7e6ef2baa